From 9714d0d4b9e170f02bf0c17914bc194efd730cbc Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 17 Feb 2015 20:42:08 +0000
Subject: [PATCH 177/462] Mantis 0034055 extension: Payment: support CCV
 OPP-C60; SCR-C

create TypeSystemHolder in directory xmlclasses.generated
for source generation from schema files

git-svn-id: svn://localhost/SelfServiceBuild/trunk@267 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 1 +
 SelfServiceBuild/Build/java/AntTargets.xml    | 4 ++--
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 33df745..8f43cdf 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -87,6 +87,7 @@
 <property name="ResourceDir"                value="resources"/>
 <property name="GenSrcDir"                  value="sources.generated"/>
 <property name="GenXMLSrcDir"               value="xmlsources.generated"/>
+<property name="GenXMLClassDir"             value="xmlclasses.generated"/>
 <property name="ClassDir"                   value="ant_classes"/>
 <property name="TestClassDir"               value="ant_junit_classes"/>
 <property name="TestSrcDir"                 value="test-sources"/>
diff --git a/SelfServiceBuild/Build/java/AntTargets.xml b/SelfServiceBuild/Build/java/AntTargets.xml
index 29f8c8e..e16ff9a 100644
--- a/SelfServiceBuild/Build/java/AntTargets.xml
+++ b/SelfServiceBuild/Build/java/AntTargets.xml
@@ -87,7 +87,7 @@
     <mkdir dir="${GenXMLSrcDir}/${GenFile}"/>
     <!-- Die angegebene Schema-Datei wird compiliert. -->
     <echo message="Compiling XML-Beans Schema: ${SchemaFile}"/>
-    <xmlbean  srcgendir="${GenXMLSrcDir}/${GenFile}" destfile="${JarDir}/${GenFile}.jar" classpath="${JarXMLBeans};${JarJSR173}">
+    <xmlbean classgendir="${GenXMLClassDir}" srcgendir="${GenXMLSrcDir}/${GenFile}" destfile="${JarDir}/${GenFile}.jar" classpath="${JarXMLBeans};${JarJSR173}">
       <fileset file="${SchemaFile}" />
     </xmlbean>
     <jar destfile="${JarDir}/${GenFile}.jar" manifest="${ManifestDir}/${ManifestName}" update="true"/>
@@ -98,7 +98,7 @@
     <mkdir dir="${GenXMLSrcDir}/${GenFile}"/>
     <!-- Die angegebene Schema-Datei wird compiliert. -->
     <echo message="Compiling XML-Beans Schema: ${SchemaDir}"/>
-    <xmlbean  srcgendir="${GenXMLSrcDir}/${GenFile}" destfile="${JarDir}/${GenFile}.jar" classpath="${JarXMLBeans};${JarJSR173}">
+    <xmlbean  classgendir="${GenXMLClassDir}" srcgendir="${GenXMLSrcDir}/${GenFile}" destfile="${JarDir}/${GenFile}.jar" classpath="${JarXMLBeans};${JarJSR173}">
       <fileset dir="${SchemaDir}" />
     </xmlbean>
     <jar destfile="${JarDir}/${GenFile}.jar" manifest="${ManifestDir}/${ManifestName}" update="true"/>
-- 
2.41.0.windows.1

